certified-connectors/OpenLegacy IBM Mainframe/apiDefinition.swagger.json

```diff
@@ -0,0 +1,769 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "OpenLegacy IBM Mainframe",
+        "description": "OpenLegacy IBM Mainframe connector provides bidirectional connectivity to your system’s data and business logic through APIs deployed on OpenLegacy’s cloud or any other target deployment. You can now build Business Flows with information from your legacy environment using the OpenLegacy connector. This will enable you to utilize your legacy assets as digital components, providing you a legacy integration and modernization solution from Day 1.",
```

Comment:
@boiv please remove or replace the word "API" in the description. please see [here ](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info).

Reply:
@Amjed-Ayoub 
The description was updated